src/db/util/raw-model.ts

```diff
@@ -172,6 +182,10 @@ export const defineRawModel =
         query.orderBy(orderBy);
       }
 
+      if (orderByRaw !== undefined && orderByRaw !== '') {
+        query.orderByRaw(orderByRaw);
+      }
```

Comment:
For raw queries, you should use `queryBuilder`, like you are doing in test file `tests/db/conditions.spec.ts`. Just remove `orderByRaw` option. When rebasing, just drop commits ef7d6ee6008cb89e4e93a42c9cd9e09bbc7281c4 and 53a68a7e4d6a21699cfcc962727586310dd34f18.